src/nft/components/profile/list/PriceTextInput.tsx

```diff
@@ -1,13 +1,67 @@
 import { Trans } from '@lingui/macro'
-import { Box } from 'nft/components/Box'
-import { Column, Row } from 'nft/components/Flex'
+import Column from 'components/Column'
+import Row from 'components/Row'
 import { AttachPriceIcon, EditPriceIcon } from 'nft/components/icons'
 import { NumericInput } from 'nft/components/layout/Input'
-import { badge, body } from 'nft/css/common.css'
+import { body } from 'nft/css/common.css'
 import { useSellAsset } from 'nft/hooks'
 import { ListingWarning, WalletAsset } from 'nft/types'
 import { formatEth } from 'nft/utils/currency'
-import { Dispatch, FormEvent, useEffect, useRef, useState } from 'react'
+import { Dispatch, FormEvent, useEffect, useMemo, useRef, useState } from 'react'
+import styled, { useTheme } from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+import { colors } from 'theme/colors'
+
+const PriceTextInputWrapper = styled(Column)`
+  gap: 12px;
+  position: relative;
+`
+
+const InputWrapper = styled(Row)<{ borderColor: string }>`
+  height: 44px;
+  color: ${({ theme }) => theme.textTertiary};
+  padding: 4px;
+  border: 2px solid;
+  border-radius: 8px;
+  border-color: ${({ borderColor }) => borderColor};
+  margin-right: auto;
+  box-sizing: border-box;
+`
+
+const CurrencyWrapper = styled.div<{ listPrice: number | undefined }>`
+  margin-right: 16px;
+  color: ${({ listPrice, theme }) => (listPrice && listPrice >= 0 ? theme.textPrimary : theme.textSecondary)};
```

Comment:
this will always be `theme.textSecondary` when `listPrice` is `0` because it gets evaluated to `false`- if this is intentional then can you remove the `>=` and make it `>` else check for `undefined` explicitly.

Reply:
I've simplified to just check listPrice